core/modules/file/file.install

```diff
@@ -169,6 +181,39 @@ function file_schema() {
     ),
   );
 
+  $schema['file_metadata'] = array(
+    'description' => 'Cache images dimensions.',
```

Comment:
This table has capacity to be used for more than image dimensions. It should be more generic but mention image dimensions as one use-case.